src/pages/home/report/ReportActionItemSingle.tsx

```diff
@@ -106,7 +106,7 @@ function ReportActionItemSingle({
     let secondaryAvatar: Icon;
     const primaryDisplayName = displayName;
     if (displayAllActors) {
-        // The ownerAccountID and actorAccountID can be the same if the a user requests money back from the IOU's original creator, in that case we need to use managerID to avoid displaying the same user twice
+        // The ownerAccountID and actorAccountID can be the same if the a user submits expense back from the IOU's original creator, in that case we need to use managerID to avoid displaying the same user twice
```

Comment:
```suggestion
        // The ownerAccountID and actorAccountID can be the same if the a user submits an expense back from the IOU's original creator, in that case we need to use managerID to avoid displaying the same user twice
```